routes/points_test.go

```diff
@@ -177,8 +177,10 @@ func TestRoutePoints(t *testing.T) {
 	t.Run("Insert", func(t *testing.T) {
 		for _, testcase := range []genericTestcase{
 			genericTestcase{"BasicPoints", "POST", `/points`, `{"member_id":1,"object_type": 2,"object_id": 1,"points": 100}`, http.StatusOK, `{"points":1000}`},
-			genericTestcase{"InvalidToken", "POST", `/points`, `{"member_id":1,"object_type": 2,"object_id": 1,"points": -100}`, http.StatusBadRequest, `{"Error":"Invalid Token"}`},
-			genericTestcase{"InvalidMemberInfo", "POST", `/points`, `{"member_id":1,"object_type": 2,"object_id": 1,"points": -100, "token": "token"}`, http.StatusBadRequest, `{"Error":"Invalid Payment Info"}`},
+			genericTestcase{"InvalidToken", "POST", `/points`, `{"member_id":1,"object_type": 3,"object_id": 1,"points": -100}`, http.StatusBadRequest, `{"Error":"Invalid Token"}`},
```

Comment:
a quick question, will we accept the transaction with 0 point?

Reply:
The only rejection is minus amount of point in top-up mode.
Other modes ( project, project-memo, gift ) accept positive, 0 and negative point value.